crates/ironclaw_gateway/static/index.html

```
@@ -414,27 +693,48 @@ <h2 data-i18n="restart.title">Restart IronClaw Instance</h2>
+            <button class="settings-subtab" data-settings-subtab="users" data-i18n="settings.users">Users</button>
+          </div>
+          <div class="settings-sidebar-spacer"></div>
           <button class="settings-theme-toggle" id="settings-theme-toggle" data-i18n="theme.tooltipSystem" title="Toggle theme">Theme</button>
         </div>
         <div class="settings-content">
           <div class="settings-toolbar">
-            <button id="settings-back-btn" class="settings-back-btn">&larr; Back</button>
-            <div class="settings-search">
-              <input type="text" id="settings-search-input" data-i18n-placeholder="settings.searchPlaceholder" placeholder="Search settings..." data-i18n-attr="aria-label" data-i18n="settings.searchPlaceholder" aria-label="Search settings...">
-            </div>
+            <button id="settings-back-btn" class="settings-back-btn">&larr; <span data-i18n="common.back">Back</span></button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)